.github/workflows/test.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       matrix:
         platform: [ubuntu-latest]
         go-version:
+          - 1.20.x
           - 1.19.x
```

Comment:
Do we need to test in all of these versions?

Reply:
No, lets remove non go-generics supported ones